src/telegram/_message.py

```diff
@@ -628,6 +628,10 @@ class Message(MaybeInaccessibleMessage):
             of a channel has changed.
 
             .. versionadded:: 22.3
+        reply_to_checklist_task_id (:obj:`int`, optional): Identifier of the specific checklist
```

Comment:
please modify `test_message_type_completeness` to not consider this attribute a separate message type :)